page.php

```diff
@@ -0,0 +1,38 @@
+<?php
+get_header();
+
+$content_two = get_post_meta($post->ID, '_igv_content_two', true);
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="margin-bottom-large padding-top-large">
+
+  <!-- main posts loop -->
+  <section id="page" class="container">
+
+    <article <?php post_class('row font-size-mid'); ?>>
+      <div class="col col-s-12 col-m-6 col-l-5">
+        <?php the_content(); ?>
+      </div>
+      <div class="col col-s-12 col-m-6 col-l-5 offset-l-1">
```

Comment:
readability. I don't think the reasoning is obvious with classes in markup but a p max width is self explanatory. At first I didn't understand this line and thought the offset was supposed to be 2 and it was a mega gutter